lib/client/Client.js

```diff
@@ -650,7 +650,7 @@ export default class Client {
     return {
       Accept: 'application/json; charset=UTF-8',
       'Content-Type': 'application/json; charset=UTF-8',
-      Authorization: this.authToken
+      Authorization: 'X-JMAP ' + this.authToken
```

Comment:
Humm... This will break other authentication mechanisms. The user is still free to do authentication without the library and use oauth for instance. We should not enforce X-JMAP here. For instance in OpenPaas Linagora is using "Bearer" because we use a custom-made JWT authentication done outisde this library.

However, as we should use X-JMAP when JMAP authentication is used, you can enfore "X-JMAP" in the header in the `withAuthAccess` method that implicitely sets the authentication token. The method is almost always certainly used right after the authentication is successful...

WDYT?